### ISSUE TITLE
Make feature and update site

### ISSUE THREAD

**Issue by khatchad** [ponder-lab/Logging-Level-Evolution-Plugin#148]
## Plug-ins to include in the feature
- logging core
- logging UI
- mylyngit core

## Feature version
For the first version, use 0.1.0.

## Update Site
Once you commit both the feature and update site "plug-ins", update the README.md file to show the raw github URL for the update site similar to https://github.com/ponder-lab/Optimize-Java-8-Streams-Refactoring#update-site.

## Notes
Please work with @saledouble.

**Comment by walleXD** [comment]
added in #149

**Comment by khatchad** [comment]
This needs to be tested.

**Comment by khatchad** [comment]
I need to see some evidence that this works, please.

**Comment by khatchad** [comment]
As discussed, @walleXD please fork this repo. Also, receiving this error:
```
Cannot complete the install because one or more required items could not be found.
  Software being installed: Feature 0.1.0 (edu.cuny.hunter.log.feature.feature.group 0.1.0)
  Missing requirement: Core 1.0.0.201902281022 (edu.cuny.hunter.log.core 1.0.0.201902281022) requires 'java.package; edu.cuny.citytech.refactoring.common.core 0.0.0' but it could not be found
  Cannot satisfy dependency:
    From: Feature 0.1.0 (edu.cuny.hunter.log.feature.feature.group 0.1.0)
    To: org.eclipse.equinox.p2.iu; edu.cuny.hunter.log.core [1.0.0.201902281022,1.0.0.201902281022]
```

**Comment by khatchad** [comment]
And then test it on a fresh workspace that does not have any other plug-ins (dependencies installed).

**Comment by khatchad** [comment]
Since the new feature will include new plugins, we will need to bump:
1. The feature version to 0.2.0.
1. The plugin versions that changed since 0.1.0. These include:
  - Core (bump to 1.1.0)
  - UI (bump to 1.1.0)

Tag the commit (try to use one commit) with the new *feature* version (0.2.0).

**Comment by khatchad** [comment]
Note also that this problem exists in other plugins of ours. See https://github.com/ponder-lab/Optimize-Java-8-Streams-Refactoring/blob/f04ceac6f827be2ace67fe4faaeda15d286763e0/edu.cuny.hunter.streamrefactoring.feature/feature.xml#L276-L302 and https://github.com/ponder-lab/Optimize-Java-8-Streams-Refactoring/blob/f04ceac6f827be2ace67fe4faaeda15d286763e0/edu.cuny.hunter.streamrefactoring.feature/feature.xml#L227.

**Comment by walleXD** [comment]
fix in #156
